src/buffer/out/search.cpp

```diff
@@ -22,14 +23,14 @@ using namespace Microsoft::Console::Types;
 // - direction - The direction to search (upward or downward)
 // - sensitivity - Whether or not you care about case
 Search::Search(IUiaData& uiaData,
-               const std::wstring_view str,

```

Comment:
Please note in the PR body that it rewrites Search to resolve glyph widths at search time rather than creation time (and not to store cells but to store the string itself)?